pallets/collator-selection/src/lib.rs

```diff
@@ -289,6 +300,16 @@ pub mod pallet {
 		NoAssociatedValidatorId,
 		/// Validator ID is not yet registered.
 		ValidatorNotRegistered,
+		/// Some doc.
+		OnInsert,
```

Comment:
more idiomatic way within our libs to name the errors is not describing on what occasion error has happened, but has happened, `FailedToUpdateCandidateList` for example

Reply:
Yes, the names are just placeholders right now. Same with doc comments. I'll write some proper ones once the other comments are addressed.

Reply:
Done.